src/dpi.rs

```diff
@@ -297,7 +297,7 @@ impl<P: Pixel, X: Pixel> From<(X, X)> for LogicalSize<P> {
 }
 
 impl<P: Pixel, X: Pixel> Into<(X, X)> for LogicalSize<P> {
-    fn into(self: LogicalSize<P>) -> (X, X) {
+    fn into(self: LogicalSize<f64><P>) -> (X, X) {
```

Comment:
Has this syntax ever been possible? :eyes:

Reply:
No, that was an error with my regexes.

Reply:
Heh, I suspected so :p